js/form.js

```diff
@@ -0,0 +1,27 @@
+const adForm = document.querySelector('.ad-form');
+const adFieldsets = adForm.querySelectorAll('fieldset');
+const mapFormFilter = document.querySelector('.map__filters');
+const mapSelects = mapFormFilter.querySelectorAll('select');
+const mapFieldsets = mapFormFilter.querySelectorAll('fieldset');
+
+const changeStateElements = (elements, isDisabled) => {
+  isDisabled ?
+    elements.forEach((element) => element.setAttribute('disabled', 'disabled')) :
+    elements.forEach((element) => element.removeAttribute('disabled'));
+};
+
+export const changeStateForm = (isVisible) => {
```

Comment:
Мисматч в нейминге. Почему visible? Мы ведь не скрываем/показываем форму, а дизейблим/активируем
```suggestion
export const changeStateForm = (isActive) => {
```